addons/content-browse/common/src/main/java/org/commonjava/indy/content/browse/ContentBrowseController.java

```diff
@@ -89,7 +92,13 @@ private ContentBrowseResult renderResult( final StoreKey key, final String reque
         String path = requestPath.endsWith( "/" ) ? requestPath : requestPath + "/";
 
         final ArtifactStore store = getStore( key );
+
+        // This is used to let galley ignore the NPMPathStorageCalculator handling,
+        // which will append package.json to a directory transfer and make listing not applicable.
+        ThreadContext context = ThreadContext.getContext( true );
+        context.put( RequestContextHelper.IS_RAW_VIEW, Boolean.TRUE );
```

Comment:
dup? line 101 also set this.

Reply:
line 101 set it to false back. I ref the PromotionManager line 666.

Reply:
Ok. I see. LGTM.